assets/css/app.css

```diff
@@ -22,11 +22,9 @@ html, body, #content-wrapper, .page-content, #view {
 
 .logo {
   display: inline;
-  width: 100%;
   max-width: 155px;
-  height: 100%;
   max-height: 55px;
-  margin-bottom: 5px;
+  padding-bottom: 5px;
```

Comment:
Why this change?

Reply:
According to codeclimate: `margin-bottom can't be used with display: inline. Severity: Minor. Found in assets/css/app.css by csslint`.

Reply:
Does that affect the display?

Reply:
It might, but I didn't see any difference.

Reply:
I believe that the way to go then would be to just remove the `margin-bottom` property.